tools/conformance/djxl_conformance.cc

```diff
@@ -60,22 +60,52 @@ bool SaveFile(const char* filename, std::vector<uint8_t> data) {
 
 struct ImageArray {
   uint32_t xsize, ysize;
+  // total amount of channels: num_color_channels + num_extra_channels
```

Comment:
can we either assert this in this code or remove the member?

Reply:
Removed it